CMakeLists.txt

```diff
@@ -377,8 +377,10 @@ if (NOT _ONEDPL_PSTL_OFFLOAD STREQUAL off)
     add_subdirectory(src)
     add_dependencies(oneDPL pstloffload)
 
-    # TODO remove explicit linkage with libpstloffload
-    if (NOT _fsycl_pstl_offload_option)
+    if (_fsycl_pstl_offload_option)
```

Comment:
Just a small question: why there are 2 conditions, on line 356 and 380?

Reply:
Good point, extra condition here can be removed.